### PR TITLE
Use a submodule to access `items` and `entities`

### PR DESCRIPTION
**Pull request by Sceat**
#551 
allowing us to have private submodules with proprietary data

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "world/floor1"]
 	path = world/floor1
 	url = ../../aresrpg/floor1.git
+[submodule "data"]
+	path = data
+	url = git@github.com:aresrpg/ares-data.git
```
Comment (DeltaEvo) [inline .gitmodules:6]: nit: why not name it `data` ?
aresrpg/data.git ?

Comment (DeltaEvo) [inline .gitmodules:6]:
```suggestion
	url = ../../aresrpg/ares-data.git
```

**Review by DeltaEvo — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -81,6 +81,14 @@ $ git submodule update --init
 $ npm install
 ```
 
+If you have access to AresRPG's proprietary data, you can run
+
+```
+git submodule set-url data git@github.com:aresrpg/ares-data-closed.git
+git submodule update --recursive --remote
+git restore .gitmodules
+```
```
Comment (DeltaEvo) [inline README.md:90]:
```suggestion
```
git -C data remote add private git@github.com:aresrpg/ares-data-closed.git
git -C data checkout private/main
```
```

File: README.md [hunk 3]
```diff
@@ -81,6 +81,14 @@ $ git submodule update --init
 $ npm install
 ```
 
+If you have access to AresRPG's proprietary data, you can run
+
+```
+git submodule set-url data git@github.com:aresrpg/ares-data-closed.git
+git submodule update --recursive --remote
+git restore .gitmodules
```
Comment (DeltaEvo) [inline README.md:89]:
```suggestion
git -C data remote add private git@github.com:aresrpg/ares-data-closed.git
git -C data checkout private/main
```

File: .gitmodules (hunk 1 above)
Comment (Sceat) [reply .gitmodules:6]: Was about to but since i prefixed each new repo with `ares-` the choice of either will impact all of them, I'll think about that

Comment (Sceat) [reply .gitmodules:6]: hum what's the reason for the relative path ?

Comment (DeltaEvo) [reply .gitmodules:6]: This permit to use the same protocol as the parent repository,
So if the ares repo was cloned with: 
- `https://github.com/aresrpg/aresrpg.git` the url used will be `https://github.com/aresrpg/ares-data.git`
- `git@github.com:aresrpg/aresrpg.git` the url used will be `git@github.com:aresrpg/ares-data.git`

**Review by DeltaEvo — APPROVED** [review]

**Pull request merged**